### DISCUSSION TITLE
Integration of the Wallet Reader SDK in my application

### DISCUSSION THREAD

**Discussion by Mah8789** [kormax/apple-home-key-reader#18] [General]
Hello everyone, I am looking for a SDK provider or SDK, which I can integrate in my application to be able to read the wallet from Apple or Google. Or need a developer who can help implement this in my ecosystem.

**Comment by kormax** [comment]
Hello.

Is this related to this repository in any way?

I see that HomeKeys are related to NFC, but it makes no sense to read them from other mobile devices, so you have to be more precise with what you would like to achieve.

**Comment by Mah8789** [comment]
Yes , thanks for your response. Looking forward to have a
detailed discussion with you.

Please suggest your convenient time.

Thanks & Regards
Mahalakshmi
